askama_shared/src/generator.rs

```diff
@@ -637,13 +641,24 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
             )),
         }?;
 
-        let mut size_hint = self.handle(ctx, body, buf, AstLevel::Nested)?;
+        buf.writeln("_did_not_loop = false;")?;
```

Comment:
Please keep changes to the code formatting in a different commit, or just leave them out, as this makes it harder to review the actual changes here.

Reply:
What are you referring to, there is no formatting change? Before you had one one branch, now you have two, so the size_hint needs to be calculated differently.

Reply:
No clue... Never mind this.